Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_command_buffer.h

```diff
@@ -86,6 +86,8 @@ class CommandQueue final : public WrappedObject<id<MTLCommandQueue>>, angle::Non
     bool isTimeElapsedEntryComplete(uint64_t id);
     double getTimeElapsedEntryInSeconds(uint64_t id);
 
+    bool isDeviceLost() const { return mIsDeviceLost.load(std::memory_order_acquire); }
```

Comment:
In ANGLE implementation GL level, "the device is lost".
In Metal backend Metal level, I think it's the command queue that's lost, not the device?

I think it's just `return mIsDeviceLost`. (use seq_cst ordering)